src/libraries/System.Net.Mail/src/System/Net/Mail/SmtpConnection.cs

```diff
@@ -160,7 +160,7 @@ private void ShutdownConnection(bool isAbort)
                             }
                         }
                         catch (IOException) { }
-                        catch (InvalidOperationException)
+                        catch (ObjectDisposedException)
```

Comment:
@scalablecory @tmat I added `ObjectDisposedException` because we throw `InvalidOperationException` from quit command and we could hide a bug.